builtin/providers/aws/resource_aws_api_gateway_method_settings.go

```diff
@@ -0,0 +1,248 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsApiGatewayMethodSettings() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsApiGatewayMethodSettingsUpdate,
```

Comment:
There is _no_ create method, because the method settings is part of the stage which we're updating. The `aws_api_gateway_stage` resource in turn has no notion of method settings so they shouldn't clash in any way.